Cargo.toml

```diff
@@ -19,11 +18,3 @@ log = "^0.4"
 flexi_logger = "0.14.8"
 rusoto_core = "0.42.0"
 rusoto_s3 = "0.42.0"
-
-[[bin]]
-name = "commp"
-path = "src/main.rs"
-
-[[bin]]
-name = "bootstrap"
```

Comment:
this is still necessary though, it has to be `bootstrap` for lambda, see also `Makefile` which bundles it into a ZIP.

Reply:
Couldn't you just rename `src/bin/lamda.rs` to `src/bin/bootstrap.rs`? I'm not sure if that's better though.

Reply:
I could, but I wanted it to be more obvious, `bootstrap` is pretty generic